src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-be.page.spec.ts

```diff
@@ -0,0 +1,180 @@
+import { ComponentFixture, async, TestBed, fakeAsync, tick } from '@angular/core/testing';
```

Comment:
filename references `cat-be`

Reply:
I didn't introduce this in this PR. but a good spot. Guess I'll fix it

Reply:
Will remove this file entirely there is a cat c spec file already. suspect it was brought in as a result of merging with develop